src/components/InfluenceChart/index.js

```diff
@@ -0,0 +1,751 @@
+    .attr('height', IMAGE_SIZE)
+    .attr('width', IMAGE_SIZE)
+    .attr('x', -IMAGE_SIZE / 2)
+    .attr('y', -IMAGE_SIZE / 2)
+
+  canvas.append('path')
+    .attr('class', 'banner')
+    .attr('style', 'stroke-width: 25;')
+    .attr('d', populatePath(
+      'M X0 Y0 L X1 Y1',
+      [{ x: -BANNER_X, y: BANNER_Y },
+        { x: +BANNER_X, y: BANNER_Y }],
+    ))
+
+  canvas.append('text')
+    .attr('class', 'name')
+    .attr('text-anchor', 'middle')
+    .attr('y', BANNER_Y)
+    .attr('dy', '0.3em')
+    .text(node => node.person.name)
+
+  return circle
+}
+
+
+const calculateNodeScale = (node: PersonNode, centerNode: PersonNode, isMouseOver: bool): number =>
+  (node.getId() === centerNode.getId() || isMouseOver ? 1.0 : 0.5)
+
+const calculateLinkPath = (link: TLink, center: PersonNode): string => {
+  const s = link.source
+  const m = link.middle
+  const t = link.target
+
+  const angle = angleRadians(t, m)
+  const nodeRadius = ((IMAGE_SIZE / 2) * calculateNodeScale(t, center, false))
+
+  const tip = radial(t, nodeRadius, angle)
+  const left = radial(tip, 20, angle + HEAD_ANGLE)
+  const right = radial(tip, 20, angle - HEAD_ANGLE)
+
+  return populatePath(
+    'M X0 Y0 Q X1 Y1 X2 Y2',
+    [s, m, tip],
+  )
+
+}
+
+
+const renderLinks = (container: Selection, graph: TGraph): Selection => {
+  const path = container.append('path')
+
+  path.classed('link', true)
+    .classed('from', (link: TLink): bool => link.source.getId() === graph.focus.getId())
+    .classed('to', (link: TLink): bool => link.target.getId() === graph.focus.getId())
+    .style('stroke-width', ARROW_WIDTH)
+    .attr('visibity', 'visible')
+    .attr('d', (link: TLink): string => calculateLinkPath(link, graph.focus))
+    .attr('id', (link: TLink): string => `${link.source.getId()}-${link.target.getId()}`)
+
+  return path
+}
+
+
+const calculateLifelinePath = (dimensions: Dimensions, timeline: Timeline, node: PersonNode): string => {
+  const TIMELINE_UPSET = 50
+
+  if (!node.person.birthDate) {
+    return ''
+  }
+  const death = node.person.deathDate ? node.person.deathDate : moment()
+
+  const birthPx = { x: timeline.scale(node.person.birthDate), y: TIMELINE_Y(dimensions.height) }
+  const bc1 = { x: node.x, y: TIMELINE_Y(dimensions.height) - TIMELINE_UPSET }
+  const bc2 = { x: birthPx.x, y: TIMELINE_Y(dimensions.height) - TIMELINE_UPSET }
+
+  const deathPx = { x: timeline.scale(death), y: TIMELINE_Y(dimensions.height) }
+  const dc1 = { x: deathPx.x, y: TIMELINE_Y(dimensions.height) - TIMELINE_UPSET }
+  const dc2 = { x: node.x, y: TIMELINE_Y(dimensions.height) - TIMELINE_UPSET }
+
+  if (birthPx.x === undefined || deathPx.x === undefined) {
+    return ''
+  }
+
+  return populatePath(
+    'M X0 Y0 C X1 Y1 X2 Y2 X3 Y3 L X4 Y4 C X5 Y5 X6 Y6 X7 Y7', [node, bc1, bc2, birthPx, deathPx, dc1, dc2, node])
+}
+
+
+const renderLifelines = (container: Selection, dimensions: Dimensions, timeline: Timeline): Selection => {
```

Comment:
Clump all the render functions together, and clump all the calculate functions above them.